packages/core/directives/polyline-point.ts

```diff
@@ -27,8 +27,8 @@ export class AgmPolylinePoint implements OnChanges {
   ngOnChanges(changes: SimpleChanges): any {
     if (changes['latitude'] || changes['longitude']) {
       const position: LatLngLiteral = <LatLngLiteral>{
-        lat: changes['latitude'].currentValue,
-        lng: changes['longitude'].currentValue
+        lat: changes['latitude'] ? changes['latitude'].currentValue : null, // before this change it will fail if the there was no change in both latitude and longitude
+        lng: changes['longitude'] ? changes['longitude'].currentValue : null // before this change it will fail if the there was no change in both latitude and longitude
```

Comment:
@tomokohn those values are checked in the `if`, I haven't found this issue, but I would suggest a more strong check like:
`if (!!changes['latitude'] || !!changes['longitude']) {`
instead of double checking those values.

Reply:
this change was made after i found a bug when only one of these parameters changes.
it fix this issue : https://github.com/SebastianM/angular-google-maps/issues/1100

Reply:
I see @tomokohn in that case the `if` is useless